TASK-6/JsonShops.cs

```diff
@@ -0,0 +1,97 @@
+using NLog;
+using TASK_6.Exceptions;
+
+namespace TASK_6;
+
+[Serializable]
+public class JsonShops
+{
+    public List<Shop> Shops { get; set; }
```

Comment:
This class looks like util class. Dont need to have Shops list here. 
Class should contain methods with param List<Shop> and work with it